.gitignore

```diff
@@ -3,3 +3,5 @@
 .idea
 .venv
 .vscode
+.pyc
```

Comment:
As I ended up with some _.pyc_ files from the search and browse packs, not sure how, but they had to be ignored.

Reply:
This one is needed in the templates as well.. it's been a awhile since I've seen any but a good protection to have.. we have it i package.yamls already so makes sense.